src/main/java/org/ibp/api/rest/location/LocationResource.java

```diff
@@ -53,20 +61,49 @@
 		return new ResponseEntity<>(this.locationService.getLocationTypes(), HttpStatus.OK);
 	}
 
-
-	@ApiOperation(value = "List locations", notes = "Get a list of locations filter by types")
+	@ApiImplicitParams({
+		@ApiImplicitParam(name = "page", dataType = "integer", paramType = "query",
+			value = "Results page you want to retrieve (0..N)"),
+		@ApiImplicitParam(name = "size", dataType = "integer", paramType = "query",
+			value = "Number of records per page.")
+	})
+	@ApiOperation(value = "List locations", notes = "Get a list of locations filter by types, favorites, abbreviations and location name.")
 	@RequestMapping(value = "/crops/{cropname}/locations", method = RequestMethod.GET)
 	@ResponseBody
-	public ResponseEntity<List<LocationDto>> listFavoriteLocations(
+	public ResponseEntity<List<LocationDto>> listLocations(
```

Comment:
Evaluate whether to also use LocationSearchRequest as parameter in this REST API